pkg/filesystem/path/trace_test.go

```diff
@@ -4,19 +4,58 @@ import (
 	"testing"
 
 	"github.com/buildbarn/bb-storage/pkg/filesystem/path"
+	"github.com/buildbarn/bb-storage/pkg/testutil"
 	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
```

Comment:
Nit: Add empty line between github.com and golang.org imports, for consistency with the rest of the code base.